.gitignore

```diff
@@ -15,3 +15,13 @@ mingw_build/
 .vscode
 cmake-build-debug/
 cmake-build-release/
+.cache
+compile_commands.json
+resources
+Testing
+test.af
+test.bk
+test.gpl
+test.rec
+test.trn
+test2.gpl
```

Comment:
It should not be required to ignore these all. Building and running should happen in a separate build directory.

Reply:
un-added test artifacts from gitignore :+1: